addons/default/visiosoft/profile-module/resources/config/permissions.php

```diff
@@ -6,4 +6,24 @@
         'write',
         'delete',
     ],
+    'education' => [
+        'read',
+        'write',
+        'delete',
+    ],
+    'education' => [
+        'read',
+        'write',
+        'delete',
+    ],
+    'education_part' => [
+        'read',
+        'write',
+        'delete',
+    ],
+    'education_part_option' => [
+        'read',
+        'write',
+        'delete',
+    ],
```

Comment:
we dont have a stream for this "education_part_option", "education" is duplicated